src/index.ts

```diff
@@ -15,18 +17,33 @@ import {
   TotalValueLockResponse,
 } from './types/response'
 
+export {
+  Chain,
+  DefiPositionResponse,
+  DefiProfitsResponse,
+  PositionType,
+  Protocol,
+}
+
 export async function getPositions({
   userAddress,
   filterProtocolIds,
   filterChainIds,
+  blockNumbers,
 }: {
   userAddress: string
   filterProtocolIds?: Protocol[]
   filterChainIds?: Chain[]
+  blockNumbers?: Record<string, number | undefined>
```

Comment:
```suggestion
  blockNumbers?: Partial<Record<Chain, number>>
```
This will achieve the same and prevent using keys that don't exist.